doc/articles/get-started-wizard.md

```diff
@@ -96,15 +100,15 @@ On Windows, you will need to install the [GTK+3 runtime](https://github.com/tsch
 On Linux, you'll need to follow the [Uno Platform](get-started-with-linux.md#setting-up-for-linux) setup instructions.
 On macOS, you'll need to follow the [Uno Platform](get-started-vsmac.md) setup instructions.
 
-#### XAML Hot Reload troubleshooting
+#### Hot Reload troubleshooting
 
-The XAML Hot reload provides a Visual Studio for Windows output window name "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.
+The Hot reload provides a Visual Studio for Windows output window name "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.
```

Comment:
```suggestion
The Hot Reload provides a Visual Studio for Windows output window named "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.
```